README.md

```diff
@@ -16,6 +16,17 @@ This git repository is laid out as a mono-repo, containing multiple independent
 
 ### Dynamical core tests
 
+Before the top-level build, make sure you have configured the authentication with user credientials and configured Docker with the following commands:
+```shell
+$ gcloud auth login
+$ gcloud auth configure-docker
+```
+
+You will also need to update the submodules to include gt4py:
+```shell
+$ git submodule update --init
+```
```

Comment:
There are submodules other than gt4py, so something like this? Not sure if we have recursive submodules, but it can't hurt. :) 
```suggestion
You will also need to ensure the git submodules are cloned and at the correct version:
```shell
$ git submodule update --init --recursive
```